components/login/SignInKey.tsx

```diff
@@ -0,0 +1,110 @@
+import { Alert, Button, Description, Label } from "@components/forms";
+import { isValidGovEmail } from "@lib/validation";
+import React, { useReducer, useState } from "react";
+import emailDomainList from "../../email.domains.json";
+import axios from "axios";
+import { useTranslation } from "next-i18next";
+import ErrorListItem from "@components/forms/ErrorListItem/ErrorListItem";
+
+interface FormAction {
+  name: string;
+  value: string;
+}
+
+const formReducer = (state: Record<string, string>, action: FormAction) => {
+  return {
+    ...state,
+    [action.name]: action.value,
+  };
+};
+
+const SignInKey = (): React.ReactElement => {
+  const [formData, setFormData] = useReducer(formReducer, {});
+  const [errorState, setErrorState] = useState({ message: "" });
+
+  const { t } = useTranslation("login");
+
+  const handleChange = (
+    event: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLTextAreaElement>
+  ) => {
+    setFormData({
+      name: event.target.name,
+      value: event.target.value,
+    });
+  };
+
+  const handleLoginSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    setErrorState({ message: "" });
+    const loginEmail = formData["loginEmail"];
+    if (isValidGovEmail(loginEmail, emailDomainList.domains)) {
+      try {
+        await axios({
```

Comment:
If both the email and the form sign-in key (form bearer token) are valid but the request for sending the temporary token failed because the server just did not succeed it does not mean that the credentials are invalid. It may be something to discuss with @connorscarolyns in case we want to display a different error.

Reply:
@jeberhardt  I think you missed my comment right above ^

Reply:
I think we'll need a different way to present that type of error, because it doesn't fit with the model of "Problems on this page". It's not a problem that the user can fix by filling in the form differently.

Reply:
Definitely and this is why I tagged @connorscarolyns :) I think it is important to not tell the user that the credentials are wrong in such case because then they could be trying to fix a problem that does not exist.